soap/src/java/com/zimbra/soap/mail/type/ActionSelector.java

```diff
@@ -153,6 +153,13 @@
     protected String tagNames;
 
     /**
+     * @zm-api-field-tag non-existent-ids
+     * @zm-api-field-description Comma-separated list of non-existent ids
+     */
+    @XmlAttribute(name=MailConstants.A_NON_EXISTENT_IDS /* nei */, required=false)
+    protected String nonExistentIds;
```

Comment:
Puzzled by this.  I'd expect this to be a ZmBoolean - to choose whether or not to return the list of non-existent IDs in the response.  Don't see how we can know the list of unknowns when constructing the request.